evm/next/documentation/migrations/add-evm-to-existing-chain.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: "Adding EVM to an Existing Chain"
+description: "Guide for integrating the EVM module into a running Cosmos chain post-genesis"
+---
+
+## Overview
+
+This guide covers the essential configuration requirements to ensure a smooth transition from a standard Cosmos chain to an EVM-compatible chain.
+
+<Info>
+The integration process focuses on two key areas: account derivation and gas token configuration. These must be set before adding the EVM module to ensure compatibility with Ethereum tooling. If you're pre-genesis and may add Cosmos EVM in the future, we strongly recommend completing steps 1 and 2 now to avoid major migrations later.
+</Info>
+
+## Prerequisites
+
+Before beginning the integration process, ensure you have:
+
+- A running Cosmos SDK chain with governance enabled
+- Access to modify the chain's configuration and codebase
+- Understanding of your chain's current account derivation setup
```

Comment:
let's link here - this is a non-obvious thing and most new people spinning up chains don't know where this is set